### DISCUSSION TITLE
Removing `{{unimplemented_inline}}` macro

### DISCUSSION THREAD

**Discussion by bsmth** [mdn/mdn-community#637] [Content] [answered]
Hi all, I've just removed the one and only occurrence of `{{unimplemented_inline}}` in the following PR:

- [x] https://github.com/mdn/content/pull/32158

__Related removals:__

- [x] https://github.com/mdn/content/pull/25039

## Translated content

Should we proceed with removing it from the translated content also?
I can count 183 matches:

```bash
# mdn/translated-content
grep -ri "unimplemented_inline" ./files | wc -l
     183
```

**Comment by caugner** [comment]
Yeah, I think the process for macros removed from content is to deprecate thm in yari and file an issue for removal in translated-content, unless the macro is specifically used in translated-content.

**Reply by bsmth** [reply, marked as answer]
OK I have the following action items:

- [x] Yari deprecation https://github.com/mdn/yari/pull/10486
- [x] Translated content issue https://github.com/mdn/translated-content/issues/18217

__edit:__ As of Monday 25th March 2024, this is now gone from both content repositories!

Thanks to everyone who helped! 👏🏻
